ood-portal-generator/sbin/update_ood_portal

```diff
@@ -27,45 +31,76 @@ while true; do
     case "$1" in
         -h|--help) usage ; exit 0 ; shift ;;
         -r|--rpm) RPM=1 ; shift ;;
+        -f|--force) FORCE=true ; shift ;;
         *) break ;;
     esac
 done
 
+
```

Comment:
For clarity maybe add comment here i.e. `# generate checksum of Apache file if it doesn't exist`